packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -673,6 +676,12 @@ export default class NewVm extends BaseComponent {
   _addVdi = () => {
     const { state } = this.state
     const { pool } = this.props
+    const resourceSet = this._getResolvedResourceSet()
+    const srsReourceSet = resourceSet.objectsByType['SR']
+    const predicate = this._getSrPredicate()
+    const srs = predicate ? filter(srsReourceSet, predicate) : srsReourceSet
+    const defaultSr = getObject(store.getState(), srs[0].$pool, resourceSet)
```

Comment:
Why the first SR's pool? I'd say the selected template's pool.

Reply:
Isn't the third param supposed to be a boolean?

Reply:
For `getObject` the third param can be object `resourceSet` because we don't have an explicit test [here](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/selectors.js#L277toL305)